src/Apps/Order/Components/TransactionSummary.tsx

```diff
@@ -92,9 +94,13 @@ export class TransactionSummary extends React.Component<
         <StackableBorderBox flexDirection="column">
           {isOfferFlow ? (
             <>
-              <Entry label="Your offer" value={offerTotal} />
+              <Entry label="Your offer" value={offerOverride} />
               {Boolean(itemsTotal) && (
-                <Entry label="List price" secondary value={itemsTotal} />
+                <Entry
+                  label="List price"
+                  secondary
+                  value={itemsTotal || offerTotal}
```

Comment:
I think this should be `value={itemsTotal || offerTotal}`? The list price should only reflect the line items list prices, not the current offer, right?